packages/migration/src/migration/index.ts

```diff
@@ -1,5 +1,5 @@
 import { globSync } from 'glob';
-import { ReplaceInFileConfig, replaceInFileSync } from 'replace-in-file';
+import { type ReplaceInFileConfig, replaceInFileSync } from 'replace-in-file';
 import type { ReplaceResult } from 'replace-in-file';
```

Comment:
```suggestion
import type { ReplaceInFileConfig, ReplaceResult } from 'replace-in-file';
import { replaceInFileSync } from 'replace-in-file';
```

wouldn't this look nicer ?

Reply:
fixed